coral/src/app/features/components/TopicDetailsModalContent.tsx

```diff
@@ -73,20 +73,30 @@ const TopicDetailsModalContent = ({
           <dd>{topicname}</dd>
         </Box.Flex>
       </Grid.Item>
-      <Grid.Item xs={2}>
-        <Box.Flex flexDirection={"column"}>
-          <Label>Topic description</Label>
-          <dd>{description}</dd>
-        </Box.Flex>
-      </Grid.Item>
-      <Box.Flex flexDirection={"column"}>
-        <Label>Topic partition</Label>
-        <dd>{topicpartitions}</dd>
-      </Box.Flex>
-      <Box.Flex flexDirection={"column"}>
-        <Label>Topic replication factor</Label>
-        <dd>{replicationfactor}</dd>
-      </Box.Flex>
+      {/*getTopicRequestsForApprover for a CLAIM request*/}
```

Comment:
I hope the comments helps future 👀  seeing this code and explain why it's dependent on "CLAIM" request. This could also be a backend fix in the future to make openapi more safe.